### DISCUSSION TITLE
Chia Network app does not connect after the update

### DISCUSSION THREAD

**Discussion by MinerNovice** [Chia-Network/chia-blockchain#7306] [Support]
Hello,

After 1.2.0 upgrade (tried both update on old one, and delete and install new one) my Chia Wallet does not start.
It is written "Connecting to wallet" and for 20minutes happens nothing. Tried to restart, did not help.
Any ideas?

![image](https://user-images.githubusercontent.com/87203585/125117895-8e11b080-e0f7-11eb-8950-bde35b7d5342.png)

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
